src/Servers/Kestrel/test/FunctionalTests/RequestTests.cs

```diff
@@ -213,6 +213,74 @@ public async Task CanHandleMultipleConcurrentRequests()
             }
         }
 
+        [Fact]
+        public async Task RequestBodyReadAsyncCanBeCancelled()
```

Comment:
Actually we don't need the tests in 2.2 as they already exist in the other project: https://github.com/aspnet/AspNetCore/blob/3b1a89c1fcb4a5b55b6303127fff4352a423974f/src/Servers/Kestrel/test/InMemory.FunctionalTests/RequestTests.cs#L62